src/ui/coordinates/coordView.ts

```diff
@@ -74,7 +42,7 @@ export default function view(ctrl: CoordCtrl) {
                 type: 'radio',
                 name: 'color',
                 value: 1,
-                onclick: () => reOriente('white'),
+                onclick: () => ctrl.chessground.orienteWithColor('white'),
```

Comment:
you can use `renderRadio` or `renderMultipleChoiceButton` from `src/ui/shared/form.ts`  instead of this.

Reply:
I used renderRadio but I have to make some changes to it.